packages/mobile/src/web3/contracts.ts

```diff
@@ -67,11 +66,13 @@ function getWebSocketProvider(url: string): Provider {
 }
 
 function getWeb3(): Web3 {
-  Logger.info(`Initializing web3, platform: ${Platform.OS}, geth free mode: ${isZeroSyncMode()}`)
+  Logger.info(
+    `Initializing web3, platform: ${Platform.OS}, geth free mode: ${isInitiallyZeroSyncMode()}`
+  )
 
-  if (isZeroSyncMode() && Platform.OS === 'ios') {
+  if (isInitiallyZeroSyncMode() && Platform.OS === 'ios') {
```

Comment:
I think zeroSync should work on iOs, but I haven't had the chance to test it yet, so leaving this as is